lib/node_modules/@stdlib/blas/ext/base/sapx/benchmark/benchmark.ndarray.native.js

```diff
@@ -22,16 +22,17 @@
 
 var resolve = require( 'path' ).resolve;
 var bench = require( '@stdlib/bench' );
-var randu = require( '@stdlib/random/base/randu' );
+var uniform = require( '@stdlib/random/base/uniform' ).factory;
+var filledarrayBy = require( '@stdlib/array/filled-by' );
 var isnan = require( '@stdlib/math/base/assert/is-nan' );
 var pow = require( '@stdlib/math/base/special/pow' );
-var Float32Array = require( '@stdlib/array/float32' );
 var tryRequire = require( '@stdlib/utils/try-require' );
 var pkg = require( './../package.json' ).name;
 
 
 // VARIABLES //
 
+var rand = uniform( -100.0, 100.0 );
```

Comment:
Move below L39.

Reply:
sorry for the styling mistakes but they are all fixed now 😅